### PR TITLE
Feature/image upload

### PR DESCRIPTION
**Pull request by pclundaahl**
Adds image upload functionality for users who are visiting their own profile.

### CODE REVIEW

File: src/client/components/profilePicture.js [hunk 1]
```diff
@@ -1,16 +1,123 @@
...
+      backgroundImage: `url(${this.state.picture})`,
+    }
+
+    return (
+      <div className={`image-wrapper-div`} style={wrapperStyle}>
+        <div className={`picture-screen ${ this.props.isEdit && 'fade-picture' }`}></div>
+        <img className={`profile-picture img-fluid`}
+            style={imgStyle}
+            src={DEFAULT_PICTURE_URL}
+            alt="">
```
Comment (cwarje) [inline src/client/components/profilePicture.js]: Would be good to have a value for the alt attribute.

File: src/server/api/routes/users.js [hunk 2]
```diff
@@ -368,4 +410,43 @@ router.delete("/:userId/by-obj-id", (req, res, next) => {
...
+        else {
+            const pictureUrl = `https://s3-${process.env.AWS_REGION}.amazonaws.com/${process.env.S3_BUCKET_NAME}/${process.env.S3_IMAGE_PATH}/${imgName}?cacheStop=${Date.now()}`;
+
+            User.update({ userId: req.params.userId },
+                {$set: { picture: pictureUrl }}).exec()
+            .then(() => {
+                res.status(200).json({ picture: pictureUrl });
+            })
+            .catch((err) => {
+                log.error('ERROR HERE 2', err);
```
Comment (cwarje) [inline src/server/api/routes/users.js]: Could use a more descriptive log here :)

Comment (pclundaahl) [reply src/server/api/routes/users.js]: Bahaha! My bad. I must have been pretty burnt out at this point.

Comment (cwarje) [reply src/server/api/routes/users.js]: Haha all good.

**Pull request merged**